packages/forma-36-website/src/content/deprecated-components/typography/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Typography'
+title: 'Typography Component'
 status: 'deprecated'
 ---
 
```

Comment:
Maybe we can add here some small description with the link on our migration guides?
> The Typography component was removed. For more information, see [the detailed migration guide](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-typography-components).

Reply:
good idea. Thanks! 🙂